btcpay-setup.sh

```diff
@@ -383,25 +448,32 @@ stop on runlevel [!2345]
 # respawn # might cause over charge
 
 script
-    . /etc/profile.d/btcpay-env.sh
+    . \"$BASH_PROFILE_SCRIPT\"
     cd \"\$BTCPAY_BASE_DIRECTORY/btcpayserver-docker\"
     . helpers.sh
     btcpay_up
 end script" > /etc/init/start_containers.conf
     echo -e "BTCPay Server upstart configured in /etc/init/start_containers.conf\n"
-if $START; then
-    initctl reload-configuration
-    echo "BTCPay Server started"
-fi
+
+	if $START; then
+		initctl reload-configuration
+		echo "BTCPay Server started"
+	fi
 fi
 
+#if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
delete itif not used

Reply:
Should I keep the TODO? Not sure why the upstart script exists. Can't Docker start the containers on its own without the upstart script?

Reply:
While docker start automatically, I don't think it starts a docker-compose service automatically.

Reply:
Just did some more testing and can confirm that the containers go back up after a reboot.